### PR TITLE
minor/feature/UBI-526-fuzzing

### PR DESCRIPTION
**Pull request by JeremiSavard**
#### What changes does this PR contain?
fuzzing for json marshal/unmarshal
#### Which JIRA ticket ID's are associated with this PR?
ubi-526

### CODE REVIEW

File: pkg/fuzzing/fuzz.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	ID          string  `json:"id" bson:"_id"`
+	Name        string  `json:"name" validate:"required"`
+	Description string  `json:"description"`
+	Price       float32 `json:"price" validate:"gt=0"`
+	SKU         string  `json:"sku" validate:"required,sku"`
+	CreatedOn   string  `json:"-"`
+	UpdatedOn   string  `json:"-"`
+}
+
+func Fuzz(data []byte) int {
```
Comment (ArcticWalrus) [inline pkg/fuzzing/fuzz.go:15]: Can you add some documentation in a readme or something in this directory on how to use the package

Like running the go-fuzz command or at least have a link to the package somewhere

**Review by ArcticWalrus — CHANGES_REQUESTED** [review]

File: pkg/fuzzing/corpus/1d9de5f154bbfe8edadbd5b13d4ffd487c2b7448-2 [hunk 2]
```diff
@@ -0,0 +1 @@
+{"":"","":7,"":""}
```
Comment (ysma500) [inline pkg/fuzzing/corpus/1d9de5f154bbfe8edadbd5b13d4ffd487c2b7448-2:1]: No return EOF

File: pkg/fuzzing/corpus/86f9d6aef7b74718b27f824cb4f855e43273da4d-1 [hunk 3]
```diff
@@ -0,0 +1 @@
+{"Name":"Jeremi","Price":7,"SKU":"abs289102309bs-abscd"}
```
Comment (ysma500) [inline pkg/fuzzing/corpus/86f9d6aef7b74718b27f824cb4f855e43273da4d-1:1]: Same here: No return EOF

File: pkg/fuzzing/corpus/acee34f457cebfeb0a2a4082b804812befb64015-1 [hunk 4]
```diff
@@ -0,0 +1 @@
+{"e":"","ce":7,"SKU":""}
```
Comment (ysma500) [inline pkg/fuzzing/corpus/acee34f457cebfeb0a2a4082b804812befb64015-1:1]: Same here: No return EOF

File: pkg/fuzzing/corpus/d1bc3451814111f1b15bf3254eed284d1c98323d-1 [hunk 5]
```diff
@@ -0,0 +1 @@
+{"Name":"Jeremi","Price":7,"SKU":"abs-abs-ªabscd"}
```
Comment (ysma500) [inline pkg/fuzzing/corpus/d1bc3451814111f1b15bf3254eed284d1c98323d-1:1]: Same here: No return EOF

**Review by ysma500 — APPROVED** [review]
Other than comments, LGTM

**Review by ArcticWalrus — APPROVED** [review]

**Pull request merged**